library/src/utils/text.ts

```diff
@@ -4,18 +4,23 @@ export const isBoolString = (str: string) => str.trim() === 'true'
 
 export const kebab = (str: string) =>
   str
-    .replace(/([a-z0-9])([A-Z])/g, '$1-$2')
-    .replace(/([a-z])([0-9]+)/gi, '$1-$2')
-    .replace(/([0-9]+)([a-z])/gi, '$1-$2')
+    .replace(/([A-Z])/g, '-$1')
     .toLowerCase()
 
-export const camel = (str: string) =>
-  kebab(str).replace(/-./g, (x) => x[1].toUpperCase())
+export const camel = (str: string) => str
```

Comment:
The dataset already did enough work of camelizing. Just pass it through. There are some differences, but it's just that the original camel function stripped more things that the user never would have put in the attribute string if they were sane and wanted camel case.

Reply:
Doesn’t this break the `__case.camel` modifier in the case of `data-class-dark-blue__case.camel`?

Reply:
I suppose your changes assume the key is coming in as dataset casing.

Reply:
Yep, all the casing mods now assume dataset casing input. (And I think I eliminated all use of them outside that)